doc/changes.rst

```diff
@@ -203,6 +203,9 @@ Changes to individual commands
 
 ``history``
   * ``undo`` subcommand now accepts ``--ignore-extras`` and ``--ignore-installed`` like original ``history replay`` command.
+  * ``store`` subcommand now outputs a directory with transaction JSON file instead of a single transaction JSON file directly.
```

Comment:
Somehow it is not clear to me. What about:
``store`` subcommand accepts directory path as an argument instead of file path. The default is ...

Reply:
I have updated it and added a new line about the `--output` option.